Don't produce *.opt-1.pyc and *.opt-2.pyc files - Kitty doesn't run Python
interpreter with -O / -OO, so they are never used, just a waste of disk
space (3.6 MiB).

--- a/setup.py
+++ b/setup.py
@@ -938,7 +938,7 @@
             pass
         compileall.compile_dir(base_path, **kw)  # type: ignore
 
-    for optimize in (0, 1, 2):
+    for optimize in (0,):  # XXX-Patched
         c(base_path, ddir='', force=True, optimize=optimize, quiet=1, workers=num_workers)
 
 
